src/App.js

```diff
@@ -2,19 +2,46 @@ import React, { Component } from 'react';
 import Movies from './Movies';
 import Navbar from './Navbar';
 import movieData from './movieData';
+import MoviePreview from './MoviePreview';
 import './App.scss';
 
 class App extends Component {
   constructor() {
     super();
-    this.state = movieData;
+    this.state = {
+      homePageMovies: movieData,
+      clickedMovie: '',
+      movieObject: {}
```

Comment:
Great work expanding out the state to include the clicked movie and adding conditional formatting to display it